Src/Library/Endpoint/Endpoint.Setup.cs

```diff
@@ -598,13 +590,7 @@ protected void Validator<TValidator>() where TValidator : IValidator
     /// <summary>
     /// specify one or more http method verbs this endpoint should be accepting requests for
     /// </summary>
-    protected void Verbs(params Http[] methods)
-        => Verbs(methods.Select(m => m.ToString()).ToArray());
-
-    /// <summary>
-    /// specify one or more http method verbs this endpoint should be accepting requests for
-    /// </summary>
-    public sealed override void Verbs(params string[] methods)
```

Comment:
we can't remove this method and constrain to the `Http` enum. This has to stay a `string[]` because there were some edge cases sometime back where people wanted to use http methods that we don't include in the enum. also if i remember correctly, some wanted to do custom verbs as well. so i don't think we can make this change.